### PR TITLE
Change JS SDK -> Web SDK

### PR DESCRIPTION
**Pull request by ethan605**
# Problem

We want to call this as `Web SDK` rather than `JS SDK`.

# Solution

Replace all _string_ values from `JS SDK` to `Web SDK` and `js-sdk` to `web-sdk`.
Notes:
- Emails are all aliased, i.e. `js-sdk@onfido.com` and `web-sdk@onfido.com` are working interchangeably.
- Woopra values are kept for backward compatibility.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1198-pr-onfido-sdk-ui-onfido.surge.sh

**Review by kopijunkie — APPROVED** [review]
LGTM in terms of documentation changes. 

Has the Woopra domains, cookie name been renamed yet or is that still TBD?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1198-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 1]
```diff
@@ -714,7 +714,7 @@ Onfido.init({
 
 Please open an issue through [GitHub](https://github.com/onfido/onfido-sdk-ui/issues). Please be as detailed as you can. Remember **not** to submit your token in the issue. Also check the closed issues to check whether it has been previously raised and answered.
 
-If you have any issues that contain sensitive information please send us an email with the ISSUE: at the start of the subject to [js-sdk@onfido.com](mailto:js-sdk@onfido.com).
+If you have any issues that contain sensitive information please send us an email with the ISSUE: at the start of the subject to [web-sdk@onfido.com](mailto:web-sdk@onfido.com).
```
Comment (stefaniacardenas) [inline README.md:717]: Has this email already been created?

**Review by stefaniacardenas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (stefaniacardenas) [reply README.md:717]: Ignore this. I've found the answer.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1198-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by ethan605** [comment]
> Has the Woopra domains, cookie name been renamed yet or is that still TBD?

For now, all strings related to Woopra will be preserved to avoid conflicts between versions. Otherwise, it's affecting internally rather than externally.

**Pull request merged**